renderer/components/UI/Sidebar.js

```diff
@@ -8,7 +8,12 @@ import BackgroundTertiary from './BackgroundTertiary'
  * <Sidebar>Some content</Sidebar>
  */
 const Sidebar = ({ ...props }) => (
-  <BackgroundTertiary as="aside" css={{ overflow: 'hidden' }} width={3 / 12} {...props} />
+  <BackgroundPrimary
+    as="aside"
+    css={{ overflow: 'hidden', 'box-shadow': '0 2px 24px 0 rgba(0, 0, 0, 0.5)' }}
```

Comment:
I'd say extract this into styled too. generally, if it's more than one prop I try to extract. what do you think?

Reply:
I'm a little in two minds about this. I tend to prefer only extracting these into separate components when it actually adds something in terms of reusability or readability. If it's just two or three one-off style props that aren't being reused elsewhere then I don't have a problem using the styled-components built in `css` prop. That is what it is for after all.

I've updated these instances in any case.